src/patternfly/components/Table/examples/table-compact-example.hbs

```diff
@@ -120,5 +120,36 @@
         {{/dropdown}}
       {{/table-td}}
     {{/table-tr}}
+
+    {{#> table-tr}}
```

Comment:
I know adding these extra rows were used to check the bug but I think you should remove these before the PR is merged.

Reply:
What's the harm in leaving them? I left them on purpose so that if this bug crops up again, we should be able to see it in the workspace.

Reply:
Okay thats fine then!!